typings/EndBug/add-and-commit/v9/action-types.yml

```diff
@@ -44,17 +44,16 @@ inputs:
     type: string
   github_token:
     type: string
-# Please check those outputs's description and set a proper type. 'string' is just set by default
 outputs:
   committed:
-    type: string
+    type: boolean
```

Comment:
I generally appreciate doing all of this! However, what stops me from reviewing is the amount of manual checking that we'd need to do to meaningfully review it. That's why I'd first like to:

* implement #2
* I thought about a feature that would be helpful with maintaining this catalog, but might be a bit controversial: for each input and output, let's have a GitHub Actions workflow that puts its description in the comment. It's usually what is enough to check to infer the type, and if we have this information locally and always fresh, it would speed up reviewing contributions to this repo. WDYT? (Not asking you to implement it, brainstorming for now)

Reply:
You mean as PR-review-comments? Might not be a bad idea. Why should that be controversial?

Reply:
I actually meant YAML comments since they may be easier to manage than GitHub comments.

Reply:
Oh, I see, then I'm not sure how much I like it.
This is mainly for reviewing PRs, isn't it?
So as PR comments it makes sense to me so that you do not need to look at the action's YAML manually.
But duplicating the description in the typing in an automated way, I'm not so sure.

Reply:
Ok, it can be a good start to have it for PRs only. I also meant being able to quickly check if the types on the main branch are correct, but you're right it's a bit itchy.